### PR TITLE
Create renovate.json

### PR DESCRIPTION
**Pull request by donmccurdy**
Adds configuration for the Renovate bot (https://docs.renovatebot.com/), initially set to update dependencies every Monday morning, U.S. Pacific Time. Major version increments will be separate PRs for testing. Non-major version increments will be bundled into a single PR, and automatically merged if the CI passes.

Additional steps required:

- [x] Enable renovate bot (https://docs.renovatebot.com/getting-started/installing-onboarding/)
- [x] Disable dependabot

### CODE REVIEW

File: .github/renovate.json
Comment (LeviPesin) [inline .github/renovate.json:13]: What does automerge do?

Comment (donmccurdy) [reply .github/renovate.json:13]: Patch and minor version updates to dev dependencies will be grouped together into a single PR and, after the CI passes, merged automatically. Without `automerge` a maintainer would need to do the merge.

**Review by Mugen87 — APPROVED** [review]

**Comment by mrdoob** [comment]
Changed it to my timezone 😇

**Comment by mrdoob** [comment]
> - [x] Disable dependabot

I've disabled this:

Screenshot 2023-03-15 at 12 06 22

And [deleted dependabot.yml](https://github.com/mrdoob/three.js/commit/50040de3a5caa31f1eb443e8a8d905853e2b812b)

That's correct?

**Comment by LeviPesin** [comment]
You should disable Dependabot version updates also.

**Comment by mrdoob** [comment]
Ops! I shared the wrong one. I did disable version updates:

Screenshot 2023-03-15 at 15 42 23

**Comment by donmccurdy** [comment]
I think that's all that is needed. 👍🏻  We should get a one-time PR on Monday that [pins dependencies](https://github.com/donmccurdy/glTF-Transform/pull/292), or we can disable that if we want. Then future PRs will keep dependencies up to date.

**Pull request merged**
